add a class and field for natural persons

duncandewhurst opened the pull request:
Closes #264

duncandewhurst committed:
add a class and field for natural persons

pindec reviewed (approved):
Looks good!

jpmckinney commented on schema/project-level/project-schema.json:
```diff
@@ -1426,6 +1435,32 @@
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "A local identifier for this person within the people array.",
```
Will the Person object always be used in a people array? If not, then this should be made more generic.

duncandewhurst replied:
Good point, probably best to allow for reuse. Fixed in https://github.com/open-contracting/infrastructure/pull/279/commits/3397485730d766c0b694496c447cb2cff6de926f

jpmckinney commented on schema/project-level/project-schema.json:
```diff
@@ -1426,6 +1435,32 @@
+        },
+        "name": {
+          "title": "Name",
+          "description": "The full name of this person.",
```
To match construction of `jobTitle`:

```suggestion
          "description": "The full name of the person.",
```

jpmckinney reviewed (commented):
Just a couple comments. We could apply #223 to the new fields (minItems on the new array, minLength on all the new strings, minProperties on the new object), or we can wait.

duncandewhurst committed:
Update schema/project-level/project-schema.json

Co-authored-by: James McKinney <26463+jpmckinney@users.noreply.github.com>

duncandewhurst committed:
make person identifier description generic

jpmckinney commented:
There's a conflict. Otherwise, please re-request review if ready for review again.

duncandewhurst committed:
Merge branch '0.9-dev' into 264-natural-persons

duncandewhurst commented:
> There's a conflict. Otherwise, please re-request review if ready for review again.

Resolved!

> Just a couple comments. We could apply #223 to the new fields (minItems on the new array, minLength on all the new strings, minProperties on the new object), or we can wait.

Let's wait until all the schema changes for the reactive disclosures are done - I've removed #223 from the 1.0 milestone and added it to the reactive disclosures project so we don't forget.
